app/services/ValidatingPageBuilder.scala

```diff
@@ -221,6 +239,7 @@ class ValidatingPageBuilder @Inject() (pageBuilder: PageBuilder){
                         acc: List[PageVertex] = Nil)(implicit stanzaMap: Map[String, Stanza]): List[PageVertex] =
```

Comment:
In method "pageGraph" is the argument dontFollowFlows necessary? I can't see the method being invoked with 4 input arguments outside of the method itself. Internally, it takes the default value of true.

Reply:
as discussed, this points to some missing code for nested sequences, will add here